sites/partners/cypress/integration/paperApplication.spec.ts

```diff
@@ -0,0 +1,162 @@
+describe("Paper Application Tests", () => {
+  before(() => {
+    cy.login()
+  })
+
+  it("navigate to paper listing form", () => {
+    cy.get(`[col-id="status"] > a`).eq(1).click()
```

Comment:
We don't want to find any page elements that depend on the DOM structure, because that can very easily change underneath us. It looks like "Accepting Applications" is a ViewItem which already has a dataTestId prop.